Rohit/HTML basics remake/form.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>form</title>
+</head>
+<body>
+    <h1>Register</h1>
+ <form> 
+    First name: <input type="text" placeholder="First name" autocomplete="given-name" required />
+    Last name: <input type="text" placeholder="Last name" autocomplete="family-name" required /><br>
+    <br>
+    <label for="gender">gender:</label> 
+    <label for="male">Male</label>
+    <input  name="gender" type="radio" value="male" />
+    <label for="female">Female</label> 
+    <input  name="gender" type="radio" value="female" />
+    <label for="others">Others</label>
+    <input  name="gender" type="radio" value="others" /> <br>
```

Comment:
Invalid `for`. Should correspond to `id`.

Reply:
Still incomplete.